README.md

```diff
@@ -155,6 +157,15 @@ For example: `yarn run:local:report ./test-data/old ./test-data/new`
     1. Click "Publish release".
 1. The release is published to the Marketplace, and a [workflow automatically](./.github/workflows/update-tags-post-release.yml) updates the major and minor tags.
 
+## Supported file formats
+
+Lexi supports the following Markdown file formats: `.md`, `.markdown`, and `.mdx`. To configure the file formats, update the`glob` input in the workflow file. For more information, see [Example workflow](#example-workflow).
```

Comment:
I think we need to split the markup language from the extension. We support the following markup languages in lexi: Markdown, Markdoc, MDX

Some of them could have overlapping file extensions (markdoc in `.md` files, for example).

The examples are great and useful though, maybe we can keep it just for "adding .mdx" example?

Reply:
Thanks @Weetbix! 🙏 
Updated in [d96c3e3](https://github.com/Rebilly/lexi/pull/367/commits/d96c3e345366c6987a33b67a8878845f1ced8658)